include/net/net_app.h

```diff
@@ -433,8 +433,8 @@ struct net_app_ctx {
 
 	/** Running status of the server. If true, then the server is enabled.
 	 * If false then it is disabled and will not serve clients.
-	 * The server is disabled by default after initialization and will need
-	 * to be enabled manually.
+	 * The server is disabled by default after initialization and needs to
+	 * be manually enabled in order the server to serve any requests.
```

Comment:
I guess repeating "the server" part is superfluous on this line, i.e. making it "in order to serve any requests." should be good.